invokeai/app/invocations/latent.py

```diff
@@ -143,9 +143,9 @@ class TextToLatentsInvocation(BaseInvocation):
     type: Literal["t2l"] = "t2l"
 
     # Inputs
-    # TODO: consider making prompt optional to enable providing prompt through a link
     # fmt: off
-    prompt: Optional[str] = Field(description="The prompt to generate an image from")
+    positive: Optional[ConditioningField] = Field(description="Positive conditioning for generation")
```

Comment:
Would `conditioning` and `unconditioning` be more accurate field names?

Reply:
I also think you may need to add some schema config for the UI so it knows what's actually required as input here (e.g. positive conditioning may be required, whereas negative conditioning may actually be optional?)

Reply:
(I'm going to refactor the schema config structure now that we've played around with this a bit and the needs are clearer. When I do that, I'll update all nodes' Config to match.)

Reply:
Ok will add schema, about rename - not sure. From code perspective it's more accurate, but users see always "positive/negative prompt", so they don't know conditional/unconditional terms)

Reply:
+1 to teaching the users the correct terminology (eg conditioning and unconditioning)

Reply:
ehhh. if you put a prompt in the “unconditioned conditioning” tensor it’s no longer “unconditioned”. i think positive and negative are better terms at this point

Reply:
In terms of perp-neg, for example, unconditioning tensor it's empty tensor, and for prompts it's positive and negative tensors. Also I combined it in one field in last commit as without it it's hard to imagine how to implement perp-neg option.